sssd_test_framework/hosts/client.py

```diff
@@ -43,6 +43,7 @@ def features(self) -> dict[str, bool]:
 
             [ -f "/usr/lib64/sssd/libsss_files.so" ] && echo "files-provider" || :
             [ -f "/usr/libexec/sssd/passkey_child" ] && echo "passkey" || :
+            man sssd.conf | grep -q "The user to drop the privileges to" && echo "non-privileged" :
```

Comment:
Add `||` at the end, like other features. If you don't do it and a test case checks whether this functionality is provided, then the test environment will fail miserably.

Reply:
Right, it would set feature value to `non-privileged :`
Fixed.